### ISSUE TITLE
Skybox Issue with The Aether(Legacy)/Quark

### ISSUE THREAD

**Issue by Leafygreens97** [Johni0702/BetterPortals#323]
When both The Aether and Quark are installed, it doesn't seem to like switching the skyboxes very well if at all, as you can see in the screenshot below (it's showing the overworld skybox):
![2019-09-05_17 45 46](https://user-images.githubusercontent.com/26660001/64393178-6807ba80-d00d-11e9-92e8-6768c20c3175.png)

As stated in my previous issue, only BP, The Aether, and Quark are installed. (and their necessary dependencies, of course)

**Comment by Johni0702** [comment]
So, does it not switch them at all or does it sometimes switch them?
Is the sky the correct one when looking through the portal and only breaks when switching or is it broken from the start?
Does this also happen without Quark but with its dependencies?

Also, please try BP 0.3.x (likely won't make any difference, just to be sure).

**Comment by Leafygreens97** [comment]
I am having trouble reproducing the issue with just these mods isolated. Even on a fresh install with just the Aether installed, the normal pinkish-purple skybox won't show. The first time I had encountered the issue, it kept flickering between the two skyboxes while near the portal in the Aether, but I haven't been able to reproduce it since. I don't even know whether to blame Quark anymore. It may just be an Aether issue, I noticed on this fresh install that the Biome was Plains everywhere in the dimension rather than the Aether Highlands except for some random tiny spots. For reference as to the color it should be, here's a picture I found off Google Images:
![aethersky](https://user-images.githubusercontent.com/26660001/64452330-5d980000-d0a3-11e9-8570-ee0464effea8.jpg)
